src/name_bazaar/ui/pages/user_offerings_page.cljs

```diff
@@ -72,9 +74,12 @@
               :tablet 8
               :mobile 16
               :floated "right"}
-             [share-buttons
-              {:url @(subscribe [:page-share-url :route.user/offerings (select-keys @route-params [:user/address])])
-               :title (str title " on NameBazaar")}]]]
+             (when (:user/address @r-route-params)
```

Comment:
You can just check here `@route-params`. Don't need to subscribe `:resolved-route-params` in this component. Therefore `:resolved-route-params` sub can be removed altogether, I don't see other usage.